arch/common/shared_irq.c

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include "include/sw_isr_common.h"
```

Comment:
> Do you mean targeting the `include` path directly and just do

Yup, exactly

Reply:
I have no preference over having it relative or targetted with `target_include_directories`, do you prefer the latter? Is the latter a standard practice for such private headers?

Reply:
Yeah, it's not completely formalized yet (and it would be great if tooling could automatically sort headers), but this has been around for a while. Based on Google's C++ coding guidelines.

https://github.com/zephyrproject-rtos/zephyr/issues/41543